cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,11 @@ class ProductslistPage {
     private proceedToCheckoutProduct: string;     
 
     constructor() {
-        this.productList ="#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-        this.proceedToCheckoutProduct ="[style*='display: block;'] .button-container > a";
+        this.productList ="#center_column> ul > li > div a[title='Add to cart']";
+	    // Here, this selector was improved in order to avoid using classes.
```

Comment:
just the `a[title='Add to cart']` will be enough, remove the justification comment